modules/calcite/src/main/codegen/config.fmpp

```diff
@@ -102,74 +101,35 @@ data: {
       "COMPUTE"
       "ASYNC"
       "QUERY"
-      "STATISTICS",
-      "REFRESH",
-      "ANALYZE",
-      "MAX_CHANGED_PARTITION_ROWS_PERCENT",
-      "TOTAL",
+      "STATISTICS"
+      "REFRESH"
+      "ANALYZE"
+      "MAX_CHANGED_PARTITION_ROWS_PERCENT"
+      "TOTAL"
 
-      # The following keywords are reserved in core Calcite,
-      # are reserved in some version of SQL,
-      # but are not reserved in Babel.
-      #
-      # Words that are commented out (e.g. "AND") are still reserved.
-      # These are the most important reserved words, and SQL cannot be
-      # unambiguously parsed if they are not reserved. For example, if
-      # "INNER" is not reserved then in the query
-      #
-      #   select * from emp inner join dept using (deptno)"
-      #
-      # "inner" could be a table alias for "emp".
-      #
-      "A"
+      # Keywords reserved by Calcite, but not required to be reserved in Ignite.
```

Comment:
can you explain a bit - what _nonReservedKeywordsToAdd_ means in practice ? How ddl\dml commands interact with them ?